ppa/archive/tests/test_eebo_tcp.py

```diff
@@ -0,0 +1,29 @@
+import os.path
+
+from eulxml.xmlmap import load_xmlobject_from_file
+
+from ppa.archive import eebo_tcp
+from ppa.archive.tests.test_models import FIXTURES_PATH
+
+TCP_FIXTURE = os.path.join(FIXTURES_PATH, "A25820.P4.xml")
+
+
+def test_eebo_tcp_page_contents():
```

Comment:
If logic for DIV2 tags is added, a good page to check is [`<PB N="7" REF="10"/>`](https://github.com/Princeton-CDH/ppa-django/blob/9f135a48569538bc60ff21a52b1cd896f9c4011c/ppa/archive/fixtures/A25820.P4.xml#L102C1-L102C21)